spec/domain/event/approver_spec.rb

```diff
@@ -244,22 +244,23 @@ def create_application
     end
 
     it 'contains all roles if none is selected' do
-      expect(approver.current_approvers).to match_array([@rl, @va1, @va2])
+      expect(approver.current_approvers).to include(@rl, @va1, @va2)
```

Comment:
Warum die Änderungen in dieser Datei und in approval_request_job_spec? Hat das mit der neuen Rollen-Fixture zu tun? Wenn ja, könntest du nicht einfach die neue Rolle auch noch in der Liste erwähnen? Die Tests waren vorher strenger.